etl/steps/data/garden/ilostat/2023-09-19/unemployment.meta.yml

```diff
@@ -0,0 +1,70 @@
+    include less than basic (less than primary education), basic (primary and lower secondary education), intermediate (upper
+    secondary and post-secondary non-tertiary education) and advanced (short-cycle tertiary education, bachelor's, master's
+    and doctoral or equivalent levels).
+  origins:
+    - producer: International Labour Organization
+      title: Education and Mismatch Indicators (EMI)
+      description: >-
+        With the aim of promoting international comparability, statistics presented on ILOSTAT are based on standard international
+        definitions wherever feasible and may differ from official national figures. This series is based on the 13th ICLS
+        definitions. For time series comparability, it includes countries that have implemented the 19th ICLS standards, for
+        which data are also available in the Work Statistics -- 19th ICLS (WORK) database. The unemployment rate conveys the
+        number of persons who are unemployed as a percent of the labour force (i.e., the employed plus the unemployed). The
+        unemployed comprise all persons of working age who were: a) without work during the reference period, i.e. were not
+        in paid employment or self-employment; b) currently available for work, i.e. were available for paid employment or
+        self-employment during the reference period; and c) seeking work, i.e. had taken specific steps in a specified recent
+        period to seek paid employment or self-employment. Data disaggregated by level of education are provided on the highest
+        level of education completed, classified according to the International Standard Classification of Education (ISCED).
+        Data may have been regrouped from national classifications, which may not be strictly compatible with ISCED. For more
+        information, refer to the Education and Mismatch Indicators (EMI) database description.
+
+
+        Classification details  Data by age includes various age bands, usually including but not limited to youth (ages 15-24),
+        prime-age workers (age 25-54), and older workers (ages 55+). Data by sex includes data on both sexes (total), men
+        (male) and women (female). Educational attainment based on the International Standard Classification of Education
+        (ISCED). Categories include less than basic (less than primary education), basic (primary and lower secondary education),
+        intermediate (upper secondary and post-secondary non-tertiary education) and advanced (short-cycle tertiary education,
+        bachelor's, master's and doctoral or equivalent levels).
+      citation_full: Education and Mismatch Indicators (EMI) - Unemployment. International Labour Organization. ILO modelled
+        estimates database, ILOSTAT. https://ilostat.ilo.org/data/.
+      attribution_short: ILOSTAT
+      version_producer: '2023-09-10'
```

Comment:
That's right. The field `version_producer` should be used if the version of the original data product is needed to identify the data product. For example, if the data product has very distinct names "Dataset blah v3", then `version_producer=v3`. But otherwise, since we already have `date_accessed` and `date_published`, `version_producer` may not be necessary. Up to you Veronika if you think the version of the data product is particularly relevant.